misago/users/management/commands/removeips.py

```diff
@@ -1,14 +1,12 @@
-
 from django.contrib.auth import get_user_model
 from django.core.management import BaseCommand
+from misago.core.utils import ANONYMOUS_IP
+from misago.users.signals import anonymize_old_ips
 
-UserModel = get_user_model()
-
+MISAGO_IP_STORE_TIME = 33300
 class Command(BaseCommand):
-    help = "Deletes all user registered IPs"
+    help =  "Anonymizes users IPs stored for longer than set in MISAGO_IP_STORE_TIME."
 
     def handle(self, *args, **options):
```

Comment:
Command should check if `MISAGO_IP_STORE_TIME` is not falsy (eg. `if not settings.MISAGO_IP_STORE_TIME`) and then display message like "Anonymization of the old user IPs is currently disabled."

Reply:
i didn't find any settings file in misago